src/pages/BudgetApp/context/AppContext.js

```diff
@@ -0,0 +1,47 @@
+import { createContext, useReducer } from "react";
+
+const AppReducer = (state, action) => {
+    switch(action.type){
+        case 'ADD_EXPENSE':
+            return {
```

Comment:
// Keywords for search:
// sum values based on common key values:
// https://stackoverflow.com/questions/59830676/how-to-unique-and-sum-the-values-based-on-the-property-key

// 1. How can you get the cost of this
// action.payload
{
  "id": "4b6fbbbe-f6c7-4781-a6e4-a0f8cf59fc5e",
  "name": "Groceries",
  "cost": 100
}


// state.expenses
[
  {
      "id": 1,
      "name": "Groceries",
      "cost": 100 // <= 2. Added into this
  },
  {
      "id": 2,
      "name": "Holiday",
      "cost": 100
  },
  {
      "id": 3,
      "name": "Clothes",
      "cost": 500
  }
]